pkg/retrieval/pb/retrieval.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+syntax = "proto3";
+
+package retrieval;
```

Comment:
Package should be called "pb" as that is the name of the directory, which would also remove the need for named import in pkg/retrieval/retrieval.go.